src/models/Ad.js

```diff
@@ -0,0 +1,117 @@
+const mongoose = require("mongoose");
+const { Schema, Types } = mongoose;
+const uniqueArrayPlugin = require("mongoose-unique-array");
+const JobTypes = require("./JobTypes");
+const {FieldTypes, MIN_FIELDS, MAX_FIELDS} = require("./FieldTypes");
+const {TechnologyTypes, MIN_TECHNOLOGIES, MAX_TECHNOLOGIES} = require("./TechnologyTypes");
+const PointSchema = require("./Point");
+
+// Defining relevant constants
+const {MONTH_IN_MS, AD_MAX_LIFETIME_MONTHS} = require("./TimeConstants");
+
+//TODO: Define indexes? (Location is indexed in Point subdocument supposedly already) Probably should index the search fields (all?)
```

Comment:
Please do not place `TODO`s in code, create issues instead.

Reply:
Tracked here https://github.com/NIAEFEUP/nijobs-be/issues/27

Reply:
(Probably a carry-over from the old development in which the communication was being done via commit comments and not PRs since there were no draft PRs 😅 )